packages/core/workers/test/cpuCount.test.js

```diff
@@ -0,0 +1,19 @@
+import assert from 'assert';
+import os from 'os';
+
+import getCores, {detectRealCores} from '../src/cpuCount';
+
+describe('cpuCount', function() {
+  it('Should be able to detect real cpu count', () => {
+    // Windows not supported as getting the cpu count takes a couple seconds...
+    if (os.platform() === 'win32') return;
```

Comment:
I removed support for windows because the test failed due to timeout which shouldn't happen.

The command windows runs is just too slow not entirely sure if there is a way to even support it without it taking seconds.